src/Isolation/Masa.BuildingBlocks.Isolation/ITenantParserProvider.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Masa.BuildingBlocks.Isolation;
+public interface ITenantParserProvider
+{
+    string Name { get;}
+
+    Task<bool> ExecuteAsync(IServiceProvider serviceProvider);
```

Comment:
Why environment called resolved, and this called execute?

Reply:
processed